src/utils/get-config.js

```diff
@@ -8,25 +8,35 @@ import CONSTANTS from '../constants';
 const PROTOCOL_RE = /^([\w-]+?)(?=\:\/\/)/;
 
 export default function (mainPath) {
-    var configDir = '';
-
     if (statSync(mainPath).isDirectory())
-        configDir = mainPath;
-    else
-        configDir = path.dirname(mainPath);
+        mainPath = path.join(mainPath, CONSTANTS.configFileName);
 
-    var configPath   = path.join(configDir, CONSTANTS.configFileName);
-    var configString = readFileSync(configPath).toString();
+    var mainDir      = path.dirname(mainPath);
+    var configString = readFileSync(mainPath).toString();
 
     var config = JSON.parse(configString);
 
-    if (!config.appPath)
-        config.appPath = mainPath;
-    else if (!isAbsolute(config.appPath))
-        config.appPath = path.resolve(configDir, config.appPath);
+    if (config.appPath && !isAbsolute(config.appPath))
```

Comment:
This condition is duplicated in https://github.com/DevExpress/testcafe-browser-provider-electron/pull/5/files#diff-ae8cda07c4f25ff93a279e308d74a7f0R23
Please refactor that.